ingestion/src/metadata/ingestion/source/pipeline/airflow/metadata.py

```diff
@@ -311,8 +333,26 @@ def _build_dag(data: Any) -> SerializedDAG:
 
         return SerializedDAG.from_json(data)
 
+    def get_owners(self, owners) -> Optional[EntityReference]:
+        try:
+            if isinstance(owners, str) and owners:
+                user = self.metadata.get_user_by_email(email=owners)
+                if user:
+                    return EntityReference(id=user.id.__root__, type="user")
```

Comment:
nit - this part

```python
user = self.metadata.get_user_by_email(email=owners)
                if user:
                    return EntityReference(id=user.id.__root__, type="user")
```
could be a separated method to make this a bit more readable and not repeat the logic